### ISSUE TITLE
Question about Model Size

### ISSUE THREAD

**Issue by z157181773** [HuajianUP/Photo-SLAM#31]
On the third page of the Supplementary Material section of the paper, Model Size is mentioned. I would like to ask if this value represents the size of the map saved after the SLAM system is completed?
![image](https://github.com/user-attachments/assets/cbe8e8a5-b2b8-4136-bab8-75bdc476ac76)

**Comment by liquorleaf** [comment]
Yes. When using default saving path, it is the size of `{scene_result_dir}/*_shutdown/ply/point_cloud/iteration_*/point_cloud.ply`
